ambry-network/src/main/java/com.github.ambry.network/SocketServer.java

```diff
@@ -66,6 +67,7 @@
   private SSLFactory sslFactory;
 
   public SocketServer(NetworkConfig config, SSLConfig sslConfig, MetricRegistry registry, ArrayList<Port> portList) {
+    this.networkConfig = config;
     this.host = config.hostName;
     this.port = config.port;
     this.numProcessorThreads = config.numIoThreads;
```

Comment:
`selectorExecutorPoolSize` can be removed from this class.

Reply:
addressed.